pkg/kubectl/describe/versioned/describe.go

```diff
@@ -3865,7 +3865,13 @@ func describePodSecurityPolicy(psp *policyv1beta1.PodSecurityPolicy) (string, er
 		w.Write(LEVEL_0, "\nSettings:\n")
 
 		w.Write(LEVEL_1, "Allow Privileged:\t%t\n", psp.Spec.Privileged)
-		w.Write(LEVEL_1, "Allow Privilege Escalation:\t%v\n", psp.Spec.AllowPrivilegeEscalation)
+		if psp.Spec.AllowPrivilegeEscalation != nil {
+			w.Write(LEVEL_1, "Allow Privilege Escalation:\t%t\n", *psp.Spec.AllowPrivilegeEscalation)
+		} else {
+			// SetDefaults_PodSecurityPolicySpec() sets this field to a pointer
+			// to bool with value true, which is why we output "true" here.
+			w.Write(LEVEL_1, "Allow Privilege Escalation:\ttrue\n", psp.Spec.AllowPrivilegeEscalation)
```

Comment:
@liggitt WDYT of this default?

Reply:
don't do client-side defaulting. omit printing this (or print `<unset>` if the server didn't return anything)

Reply:
done.